config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+   get '/home', to: 'home#index'
+   root 'home#index'
+
+  resources :drivers
+
+  resources :trips
+
```

Comment:
Do you need all 7 restful routes for `:trips`? I think that both `:new` and un-nested `:create` are not used.